.ci/dockerfiles/Dockerfile.gpu-test

```
@@ -0,0 +1,36 @@
+ARG PRE_INSTALLED_NIXL_ENV
+ARG PRE_INSTALLED_UCX_ENV
+ARG UCX_VERSION=v1.20.x
+ARG HAS_GPU=true
+ARG WORKSPACE=/workspace/nixl
+
+
+WORKDIR ${WORKSPACE}
+COPY --chown="${_UID}":"${_GID}" . .
+
+RUN .gitlab/build.sh ${NIXL_INSTALL_DIR}
+RUN sudo apt-get clean && \
```

Hadolint (2.14.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)